src/legacy/core_plugins/visualizations/public/np_ready/public/embeddable/visualize_embeddable.ts

```diff
@@ -72,13 +74,33 @@ export interface VisualizeOutput extends EmbeddableOutput {
 
 type ExpressionLoader = InstanceType<ExpressionsStart['ExpressionLoader']>;
 
+function getOutput(
+  input: VisualizeInput,
+  { savedVisualizations, editable }: VisualizeEmbeddableConfiguration
+): VisualizeOutput {
+  const vis = new Vis(input.visObject.type, input.visObject);
+  const indexPattern = vis.data.indexPattern;
+  const indexPatterns = indexPattern ? [indexPattern] : [];
+  const editUrl = input.visObject.id
+    ? getHttp().basePath.prepend(`/app/kibana${savedVisualizations.urlFor(input.visObject.id)}`)
+    : '';
```

Comment:
in the future we should still be able to provide url here even if visualization is not saved (going to edit non saved visualization then)